e2e/terraform/packer/ubuntu-jammy-amd64/setup.sh

```diff
@@ -70,49 +73,31 @@ distro=$(lsb_release -si | tr '[:upper:]' '[:lower:]')
 curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
 sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/${distro} $(lsb_release -cs) stable"
 
-# Java
-sudo add-apt-repository -y ppa:openjdk-r/ppa
-
-# Podman
-. /etc/os-release
-curl -fsSL "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/Release.key" | sudo apt-key add -
-sudo add-apt-repository "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /"
-
-sudo apt-get update
-
+# Docker
 echo "Installing Docker"
 sudo apt-get install -y docker-ce
 
+# Java
 echo "Installing Java"
+sudo add-apt-repository -y ppa:openjdk-r/ppa
```

Comment:
Did we mean to include this? Should we remove the other JDK if so?

Reply:
Turns out I don't think we need the PPA for this anymore - jammy has openjdk 17 packaged already